package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "component-generator",
+  "version": "1.0.0",
+  "description": "Generate React components using JS or TS",
+  "main": "index.js",
+  "type": "module",
+  "bin": {
+    "component-generator": "index.js"
+  },
+  "scripts": {
+    "plop": "plop"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pagesource/component-generator.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/pagesource/component-generator/issues"
+  },
+  "homepage": "https://github.com/pagesource/component-generator#readme",
+  "dependencies": {
+    "minimist": "^1.2.5",
```

Comment:
@khus29 Please put them as peer dependencies. Since we want these packages to be updated from parent application.